core/src/main/scala/io/finch/EffectInstances.scala

```diff
@@ -0,0 +1,9 @@
+package io.finch
+
+import cats.effect.Effect
+
+trait EffectInstances[F[_]] {
```

Comment:
Given we rename `Finch` to `Module`, can we call this `Component[F[_]]`? This way a module would look like:

```scala
abstract class Module[F](implicit effect) extends Component[F[_]] with Endpoints with Boies with ect
```

Reply:
Honestly, it sounds way too generic for something that just guarantees the `Effect` instance for type `F[_]`

Reply:
Yeah, you're right. How about `ModuleComponent` or even `Module.Component`. I feel like the name should reflect that fact we use it to build a `Module` out of it.